Source/WebCore/html/PermissionsPolicy.cpp

```diff
@@ -246,21 +289,21 @@ bool PermissionsPolicy::Allowlist::matches(const SecurityOriginData& origin) con
     }), m_origins);
 }
 
-PermissionsPolicy::Allowlist PermissionsPolicy::parseAllowlist(StringView value, const SecurityOriginData& containerOrigin, const SecurityOriginData& targetOrigin, bool useStarAsDefaultAllowlistValue)
+static PermissionsPolicy::Allowlist parseAllowlist(StringView value, const SecurityOriginData& containerOrigin, const SecurityOriginData& targetOrigin, bool useStarAsDefaultAllowlistValue)
 {
     if (value.isEmpty()) {
         if (useStarAsDefaultAllowlistValue)
-            return Allowlist { Allowlist::AllowAllOrigins };
+            return PermissionsPolicy::Allowlist { PermissionsPolicy::Allowlist::AllowAllOrigins };
```

Comment:
Sure, it could be a private static method.
That said, you can probably simply write it as:
`return { PermissionsPolicy::Allowlist::AllowAllOrigins };`